guides/Git Shopify.md

```diff
@@ -0,0 +1,50 @@
+# Git strategy
+
+Our quick guide to working collaboratively with Git, GitHub and Shopify projects.
+
+## Create a Feature
+
+Check if `staging` branch present in repo.
+
+```
+git checkout master
+git pull
+git checkout staging
+```
+
+If it is present, work the same as always, but make Pull Requests on `staging`
+
+If not, then create and push it.
+
+```
+git checkout master
+git pull
+git checkout -b staging
+git push origin staging
+```
+
+## Production hotfix
+
+If you need to make a hotfix on production, create a new, local branch off `master`.
+
+```
+git checkout master
+git pull
+git checkout -b ${branch_name} # (* - check fusnote)
```

Comment:
I don't think we need to use bash variables here, nor the footnote. The following should do:

`git checkout -b hotfix-some-bug`